data/configs/org.blueman.Manager.service.in

```diff
@@ -0,0 +1,4 @@
+[D-BUS Service]
```

Comment:
Why?

Reply:
So that it can be autostarted :stuck_out_tongue_winking_eye:. I think I don't need it anymore how it is working, I'll check.

Reply:
Yes it is needed. We have to be able to start/stop it over dbus for StandardItems. And the only way is to add a dbus service.